internal/app/service/user_login_service.go

```diff
@@ -0,0 +1,59 @@
+package service
+
+import (
+	"context"
+
+	"os"
+	"time"
+
+	"github.com/golang-jwt/jwt"
+	"github.com/joshsoftware/profile_builder_backend_go/internal/pkg/errors"
+)
+
+type UserLoginServive interface {
+	GenerateLoginToken(ctx context.Context, email string) (string, error)
+}
+
+func (userService *service) GenerateLoginToken(ctx context.Context, email string) (string, error) {
+	if len((email)) == 0 {
+		return "", errors.ErrEmptyPayload
+	}
+
+	userId, err := userService.UserLoginRepo.GetUserIdByEmail(ctx, email)
+	if err != nil || userId == 0 {
+		return "", err
+	}
+
+	token, err := userService.createToken(userId, email)
+	if err != nil {
+		return "", err
+	}
+	return token, nil
+
+}
+
+func (userLoginService *service) createToken(userId int64, email string) (string, error) {
```

Comment:
create new folder in pkg and move this following functions
- createToken
- createClaims

Reply:
moved two function createToken and the createClaims function in the pkg folder